app/apollo/resolvers/index.js

```diff
@@ -22,6 +22,7 @@ This allows, for example, replacing the built in 'user' resolver with a custom o
 const resolversMap = {
   'user': './user',
   'resource': './resource',
+  'resource.updateClusterResources': './resource.updateClusterResources',
```

Comment:
I think it would make more sense for the cluster resources handling to be part of the existing `./cluster` rather than a new file with an unusual naming pattern.  Or perhaps even better, as a new `Mutation` section in `./resource.js`, where `Query` and `Subscription` on resources is already defined.  Is there a reason to keep it separate and so 'specialized', e.g. so it can be overridden in some scenario?